jobs/model_create.rb

```diff
@@ -0,0 +1,53 @@
+require 'resque'
+require 'resque-status'
+require 'sinatra/activerecord'
+require 'redis'
+require_relative './reload_cache.rb'
+require_relative '../models/videos'
+require_relative '../models/articles'
+require_relative '../models/feeds'
+require_relative '../models/resources'
+
+module Jobs
+    module Models
+        class Create
+            include Resque::Plugins::Status
+            
+            #@queue = :models
+            
+            def perform
+                # Find object in database.
+                begin
+                    @type = options['model_type'].split('::').reduce(Module, :const_get)
```

Comment:
Should document what is going on here since this is dynamic programming which is hard to follow outside context.